site2/docs/concepts-messaging.md

```diff
@@ -128,6 +128,8 @@ In the exclusive and failover subscription modes, consumers only negatively ackn
 
 In the shared and Key_Shared subscription modes, you can negatively acknowledge messages individually.
 
+Be aware that doing negative ack on ordered subscription type like Exclusive, Failover and Key_Share can cause failed messages arrive consumer out of original order.
```

Comment:
```suggestion
Be aware that negative acknowledgment on ordered subscription types, such as Exclusive, Failover, and Key_Share, can cause failed messages to arrive consumers out of the original order.
```

Reply:
`Key_Share` --> `Key_Shared`

Reply:
The same name is in many other places in this PR.

Reply:
fixed.